### ISSUE TITLE
how to subscribe with cleansession: false

### ISSUE THREAD

**Issue by kiranpradeep** [keesschollaart81/CaseOnline.Azure.WebJobs.Extensions.Mqtt#39]
The ManagedClient of MQTTNet doesn't support clean session as false. How can use cleansession:false with this extension?

**Comment by keesschollaart81** [comment]
The example project has an example of a 'Complex Configuration' as demonstrated [here](https://github.com/keesschollaart81/CaseOnline.Azure.WebJobs.Extensions.Mqtt/blob/master/src/ExampleFunctions/AdvancedConfig/ExampleMqttConfigProvider.cs#L22). Here you can add a `.WithCleanSession(true)`

**Comment by kiranpradeep** [comment]
No. It still throws "_System.NotSupportedException: The managed client does not support existing sessions_". It is an issue with underlying library. But then, users of this extension are impacted and ideally I felt this ticket should be left open until underlying library (mqtt.net) brings the [PR](https://github.com/chkr1011/MQTTnet/pull/836) to a release nuget.

**Comment by keesschollaart81** [comment]
Yeah, let's leave it open. As soon as there is a major new release of MQTTNet, I'll incorporate it so that it will work here as well! 👍
